static/templates/compose.hbs

```diff
@@ -31,11 +31,19 @@
                 </button>
             </span>
             {{/unless}}
+            {{#if is_web_public_visitor}}
+            <span class="new_message_button">
+                <a class="no-underline button small rounded float-left" href="/login" title="{{t 'Login in' }} (d)">
+                    {{t 'Login in to start conversation' }}
```

Comment:
Should we just have this replace the entire bottom-compose area?

I'd have "Log in to send a message" as the text.

Reply:
I actually find that most websites display the text box and then when you click on send button, they want you to login. I think we should wait to get more feedback on this.

Reply:
Yeah, that's reasonable logic.  The main thing I'm worried about with the current model is that the right sidebar (where we currently have the main advertisement to login) doesn't appear in some window widths.  

I think it's fine to run with this for now and iterate from there.